lua/wikis/commons/Infobox/Faction.lua

```diff
@@ -0,0 +1,124 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Infobox/Faction
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Flags = require('Module:Flags')
+local Json = require('Module:Json')
+local Lua = require('Module:Lua')
+local Namespace = require('Module:Namespace')
+
+local BasicInfobox = Lua.import('Module:Infobox/Basic')
+
+local Widgets = require('Module:Widget/All')
+local Cell = Widgets.Cell
+local Header = Widgets.Header
+local Title = Widgets.Title
+local Center = Widgets.Center
+local Customizable = Widgets.Customizable
+
+---@class FactionInfobox: BasicInfobox
+local FactionInfobox = Class.new(BasicInfobox)
+
+---@param frame Frame
+---@return Html
+function FactionInfobox.run(frame)
```

Comment:
imho move it to a /Custom (on commons)
we never should invoke the bases only ever the /Custom versions